controllers/space/space_controller_test.go

```diff
@@ -1988,6 +1988,52 @@ func TestSubSpace(t *testing.T) {
 				HasConditions(nstemplatetsettest.Provisioned())
 		})
 
+		t.Run("create multi level bindings tree", func(t *testing.T) {
+			// given a parentSpace...
+			parentSpace := spacetest.NewSpace(test.HostOperatorNs, "parentSpace")
+			// ...and their corresponding space bindings
+			sb1 := spacebindingtest.NewSpaceBinding("john", parentSpace.Name, "admin", "signupJohn")
+
+			// ...now create a subSpace
+			subSpace := spacetest.NewSpace(test.HostOperatorNs, "subSpace",
+				spacetest.WithTierName(base1nsTier.Name),
+				spacetest.WithSpecTargetCluster("member-1"),
+				spacetest.WithStatusTargetCluster("member-1"), // already provisioned on a target cluster
+				spacetest.WithSpecParentSpace(parentSpace.Name),
+				spacetest.WithFinalizer())
+			subNStmplSet := nstemplatetsettest.NewNSTemplateSet(subSpace.Name,
+				nstemplatetsettest.WithReadyCondition(),
+			)
+
+			// ... and now create a subSubSpace
+			subSubSpace := spacetest.NewSpace(test.HostOperatorNs, "subSubSpace",
+				spacetest.WithTierName(base1nsTier.Name),
+				spacetest.WithSpecTargetCluster("member-1"),
+				spacetest.WithStatusTargetCluster("member-1"), // already provisioned on a target cluster
+				spacetest.WithSpecParentSpace(subSpace.Name),
+				spacetest.WithFinalizer())
+			subSubNStmplSet := nstemplatetsettest.NewNSTemplateSet(subSubSpace.Name,
+				nstemplatetsettest.WithReadyCondition(),
+			)
+
```

Comment:
let's create a bit more complex scenario of the SpaceBindings to make sure that the overrides work as expected:
```go
spacebindingtest.NewSpaceBinding("john", parentSpace.Name, "admin", "signupJohn")
spacebindingtest.NewSpaceBinding("jane", parentSpace.Name, "edit", "signupJane")
spacebindingtest.NewSpaceBinding("bob", parentSpace.Name, "view", "signupBob")
spacebindingtest.NewSpaceBinding("lara", parentSpace.Name, "view", "signupLara")

spacebindingtest.NewSpaceBinding("jane", subSpace.Name, "admin", "signupJane")
spacebindingtest.NewSpaceBinding("lara", subSpace.Name, "edit", "signupLara")

spacebindingtest.NewSpaceBinding("bob", subSubSpace.Name, "edit", "signupBob")
spacebindingtest.NewSpaceBinding("lara", subSubSpace.Name, "admin", "signupLara")
```
expecting the final set of bindings:
john - admin
jane - admin
lara - admin
bob - edit

Reply:
Done in https://github.com/codeready-toolchain/host-operator/pull/843/commits/5f61764bd52ee77f118152ad745e64bdab25ab9e 

I was a bit unsure if we should keep using the `tiertest.CurrentBase1nsTemplates` also for this test or create a new one. For now I've added the `edit` role to the existing tier.